notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: i and j can index any one of the basis vectors (they are not the basis vectors themselves, just their indices)</p><p> </p><p>NIT: einstein -&gt; Einstein</p><p> </p><p>Nice explanation of Einstein notation!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>:D</p>